GoogleForCreators/web-stories-wp
# Removed rogue animations

## User-facing changes

"Fashion on the go" template now works correctly.

## Testing Instructions

This PR can be tested by following these steps:

1. Use the "Fashion on the go" template from "explore template"
2. Navigate through all the pages and observe no errors

## Checklist




---

Fixes #12946
